frontend/src/modules/history/components/Translation.css

```diff
@@ -113,3 +114,23 @@
 .translation > header button.unreject:hover:before {
     font-weight: 400;
 }
+
+.translation > header button.delete {
+    display: none;
```

Comment:
I'm not a big fan of hiding components with CSS. It's a form of logic imho, and I like to keep logic in the JS code. I'd rather you add a condition in the component to not show the button if the user can't reject, rather than having to go into CSS to understand when the button is shown or not.